Question on establishing a FU

alyabolowich opened the pull request:
Clarification needed on when to use get_activity() or get().

michaelweinold commented:
Hi, @alyabolowich - thank you for your contribution!

In order for your new markdown file `source/content/lca/calculate_lca.md` to be rendered in the documentation, you will need to add it to the table of content of the `source/content/lca/index.md` file:

```
---
hidden:
maxdepth: 2
---
self
static_lca
stochastic_lca
calculate_lca
```

As for the content, perhaps @cmutel can have a look?

alyabolowich commented:
Hi Michael,

I modified source/content/lca/index.md but made the mistake of pushing to
main (I'm new, sorry!). I realized the error this morning, I should have
made a new pull request for you. I will go ahead and delete what I added on
main yesterday and make a new pull request with the forked repo.

Sorry for the problems!


On Wed, Aug 9, 2023 at 12:29 PM Michael Weinold ***@***.***>
wrote:

> Hi, @alyabolowich <https://github.com/alyabolowich> - thank you for your
> contribution!
>
> In order for your new markdown file source/content/lca/calculate_lca.md
> to be rendered in the documentation, you will need to add it to the table
> of content of the source/content/lca/index.md file:
>
> ---
> hidden:
> maxdepth: 2
> ---
> self
> static_lca
> stochastic_lca
> calculate_lca
>
> As for the content, perhaps @cmutel <https://github.com/cmutel> can have
> a look?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/brightway-lca/brightway-documentation/pull/118#issuecomment-1671070864>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM4PM2RBBOW42Q2P3Y3YWDTXUNRCNANCNFSM6AAAAAA3JYYKCM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Alya Bolowich

michaelweinold commented:
> but made the mistake of pushing to main

You did not push to the `main` branch of this repo (you don't have permission to do that). You pushed to the `main` branch of your fork of this repo (`alyabolowich:main`). This is fine - you correctly opened a pull request using your branch. No worries!

> I will go ahead and delete what I added on main yesterday and make a new pull request with the forked repo.

There is no need for this. You can continue to work on the `main` branch of your fork and keep making commits. They will all show up in this pull request.

Just ping me (@michaelweinold) when you're ready 👏

alyabolowich commented:
Hi @michaelweinold,
I added the file to the TOC now. It should be OK. I will make further pull requests if I make modifications to that file.

Thanks! :)

michaelweinold commented:
Ok, thanks. In order to make the best use of our time I suggest that you keep working on that file on your own fork for now - as soon as you feel it's complete (for now), ping me to merge your PR.

michaelweinold commented:
Hello, hello! After some time, I was able to have a close look at the pending issues and pull requests in the documentation repo.

I believe it makes sense to open your two questions to discussion in the Discussion tab. I have taken the liberty to move them there:

https://github.com/brightway-lca/brightway-documentation/discussions/123
https://github.com/brightway-lca/brightway-documentation/discussions/124

As soon as we've established satisfactory answers, we can open a PR with the appropriate information. 

I'm still working on improving the contributions-workflow of the documentation (as well as the underlying structure). This should be complete by [the Brightcon conference](http://brightcon.link) (are you attending, seeing as you're base in Luxembourg?).

alyabolowich commented:
Hello,
OK great - thanks for the update. 
Yes, I'll see you at Brightcon!
